### PR TITLE
Add Outreachy mentor projects

### PR DESCRIPTION
**Pull request by hackygolucky**

### CODE REVIEW

**Comment by mikeal** [comment]
+1

File: locale/en/foundation/index.md [hunk 1]
```diff
@@ -88,10 +85,10 @@ email to <a href="mailto:trademark@nodejs.org?subject=Trademark">trademark@nodej
 		<div class="response" id="mce-error-response" style="display:none"></div>
 		<div class="response" id="mce-success-response" style="display:none"></div>
 	</div>    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
-    <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_c7c2e114a827812354112c23b_e842100924" tabindex="-1" value=""></div>
+    <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_c7c2e114a827812354112c23b_f006b61f29" tabindex="-1" value=""></div>
     <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button"></div>
     </div>
 </form>
 </div>
 <script type='text/javascript' src='//s3.amazonaws.com/downloads.mailchimp.com/js/mc-validate.js'></script><script type='text/javascript'>(function($) {window.fnames = new Array(); window.ftypes = new Array();fnames[0]='EMAIL';ftypes[0]='email';fnames[1]='FNAME';ftypes[1]='text';fnames[2]='LNAME';ftypes[2]='text';}(jQuery));var $mcj = jQuery.noConflict(true);</script>
```
Comment (lpinca) [inline locale/en/foundation/index.md:93]: The inline script will not work as the content is escaped when converted from markdown to html.

Comment (lpinca) [reply locale/en/foundation/index.md:93]: Oh sorry, didn't notice it wasn't part of the actual change.

Comment (Trott) [reply locale/en/foundation/index.md:93]: HTML is valid markdown. So it shouldn't be escaped. (And it appears that it is not.)

**Comment by Trott** [comment]
LGTM

**Review by Trott — APPROVED** [review]

**Comment by jasnell** [comment]
LGTM

File: locale/en/foundation/index.md (hunk 1 above)
Comment (lpinca) [reply locale/en/foundation/index.md:93]: @Trott not an issue on this pr, but open https://nodejs.org/en/foundation/ and see the dev tools. Also see the generated HTML on #932.

Comment (Trott) [reply locale/en/foundation/index.md:93]: Ah, yes, the quotation marks are escaped and the other things noted in that issue. Thanks for pointing that out. I would say that's a bug in the processor and not in the markdown file, but either way, yeah, needs to be addressed.

**Pull request merged**